src/test/scala/org/ergoplatform/network/ModeFeatureSpecification.scala

```diff
@@ -0,0 +1,27 @@
+package org.ergoplatform.network
+
+import org.ergoplatform.utils.ErgoPropertyTest
+
+class ModeFeatureSpecification extends ErgoPropertyTest {
+
+  property("serialization roundtrip") {
+    forAll(modeFeatureGen) {mf =>
```

Comment:
we have a separate `SerializationTests` class, this test should be there.

Reply:
fixed